packages/ndla-ui/src/AudioPlayer/AudioPlayer.js

```diff
@@ -36,7 +36,7 @@ const AudioPlayer = ({ type, src, title, typeLabel, speach }) => {
       <h1 {...classes('title')}>{title}</h1>
       <audio type={type} src={src} title={title} preload="metadata" />
       <div {...classes('controls')}>
-        <button {...classes('play')}>
+        <button disabled {...classes('play')}>
```

Comment:
Var kanskje ikke meningen å ta med dette? Den må vel komme inn som en prop isåfall.